Sources/Public/Animation/LottieAnimation.swift

```diff
@@ -157,4 +157,7 @@ public final class LottieAnimation: Codable, DictionaryInitializable {
   /// Markers
   let markers: [Marker]?
   let markerMap: [String: Marker]?
+
+  /// DotLottie configuration to setup the player
+  var dotLottieConfiguration: DotLottieConfiguration?
```

Comment:
Having this var helps on removing logic using the animations in the dotLottie file. It compiles information from manifest, image provider and DotLottieAnimation. Alternative I can think of would be to have this configuration in a separate dictionary inside DotLottieFile and then besides searching for the animation with id, also search for the configuration with id.

Reply:
> have this configuration in a separate dictionary inside DotLottieFile and then besides searching for the animation with id, also search for the configuration with id.

That sounds reasonable, I think this would be better